dtreeviz/forests.py

```diff
@@ -0,0 +1,246 @@
+import numpy as np
+import pandas as pd
+
+import matplotlib.patches as patches
+import matplotlib.pyplot as plt
+from matplotlib.collections import PatchCollection
+from colour import Color
+from PIL import ImageColor
```

Comment:
how did you manage to install PIL ? I have this error "ERROR: No matching distribution found for PIL"... 
maybe we should add it into setup.py (same for rfpimp from notebook)

Reply:
oh right. yeah, that is weird stuff. I think it's actually `Pillow` not `PIL` to install. Can you try that package?

Reply:
actually I think they are different. Hmm...

Reply:
Yep, different but we still want Pillow:

```
beast:RF-classifier-regions:~/github/dtreeviz $ pip uninstall PIL
WARNING: Skipping PIL as it is not installed.
beast:RF-classifier-regions:~/github/dtreeviz $ pip install -U Pillow
Requirement already satisfied: Pillow in /Users/parrt/opt/anaconda3/lib/python3.8/site-packages (8.1.0)
```

Reply:
I updated setup.py and pushed a new commit.